README.md

```diff
@@ -43,33 +43,26 @@ Using fluent-plugin-logdna, you can send the logs you collect with Fluentd to Lo
 </match>
 ~~~~~
 
-* Restart fluentd to pick up the configuration changes.
-* `sudo /etc/init.d/td-agent stop`
-* `sudo /etc/init.d/td-agent start`
+## Line Parameters
 
-### Options
-
-* App name and log level can be specified per each line via JSON
-* `_app` and `level` will override the config
-
-If you don't have a LogDNA account, you can create one on [https://logdna.com](https://logdna.com) or if you're on macOS w/[Homebrew](https://brew.sh) installed:
-
-~~~~~bash
-brew cask install logdna-cli
-logdna register <email>
-# now paste the api key above
-~~~~~
+The following line parameters can be set to the information coming from each `record` object:
+- `level`: [Level](https://github.com/logdna/logger-node#supported-log-levels): *Optionally* Set to either `level` or `severity` or the Last `tag` Given in Each `record`
+- `file`: File Name: *Optionally* Set to `file` Given in Each `record`
```

Comment:
Why "optionally" as opposed to "optional"

Reply:
@johnpadilla, actually, realized no need for anything like that since those are coming from `record` and can't be specified manually on line level.